CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Protocol buffers for hardware wallet transactions are no longer supported. Internet Computer Ledger app 2.4.9 or later is now required.
 - GovernanceCanister.listNeurons no longer throws an error when called with `certified: false` for hardware wallet transactions.
+- The field `smart_contract_address` was removed from ckETH minter information, and additional fields related to ERC20 support have been added.
```

Comment:
I've been told that the field `smart_contract_address` will be restored but marked deprecated instead to make this less breaking. Do you think it's worth keeping in ic-js for now?

Reply:
Discussed offline. For simplicity reason we are not going to keep it.